.evergreen/config/functions.yml

```diff
@@ -158,12 +158,17 @@ functions:
     - command: shell.exec
       params:
         script: |
-          VARIANT=${VARIANT} DRIVERS_TOOLS="${DRIVERS_TOOLS}" sh ${DRIVERS_TOOLS}/.evergreen/atlas_data_lake/build-mongohouse-local.sh
```

Comment:
The `VARIANT` variable is not necessary, or is it exported?

Reply:
It was necessary to compile on the correct platform. It's no longer necessary so we can do without :)